docs/releases/5.0.txt

```diff
@@ -170,13 +170,20 @@ types <field-choices-enum-types>`::
     }
 
 
-    class Winners(models.Model):
+    def get_scores():
+        return [(i, str(i)) for i in range(10)]
```

Comment:
Following on from my comment above about caching. If that works (without affecting the migrations stuff) then we could highlight that here?

Reply:
This does work indeed, though I would not add that detail in this What's New release note (I would not want to obfuscate this section with specific use cases).

Down below the user is invited to read the reference about model field's if they wish to know more, and since I'll be expanding that section as discussed in previous comments, I think we have the caching covered.